configure/contributing-docs.md

```diff
@@ -98,7 +98,7 @@ For titles use "Sentence case capitalization". Sentence case capitalization is m
 
 ##### Spelling
 
-Use United States English. For example, behavior rather than behavior, and referring rather than referring.
+Use United States English. For example, behavior rather than behaviour, and canceled rather than cancelled.
```

Comment:
canceled

Reply:
Resolved separately to this conversation.